cypress/integration/creatorOnboardingFlows/creatorSignup.spec.js

```diff
@@ -0,0 +1,100 @@
+describe('Creator Signup Page', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.visit('/enter?state=new-user');
+  });
+
+  it('should display a welcome message', () => {
+    cy.findByText("Let's start your Forem journey!").should('be.visible');
+  });
+
+  it('should display instructions', () => {
+    cy.findByText('Create your admin account first.').should('be.visible');
+    cy.findByText("Then we'll walk you through your Forem setup.").should(
+      'be.visible',
+    );
+  });
+
+  it('should display a validated username hint that correlates to the name entered', () => {
+    cy.findByTestId('creator-signup-form').as('creatorSignupForm');
+    cy.get('@creatorSignupForm')
+      .findByText(/^Name$/)
+      .type('1 Forem creator name! Also test a maximum length string');
+
+    // restricts the string to 20 characters, lowercased,
+    // replaces spaces and special_characters with an underscore
+    cy.contains('1_forem_creator_name__also_tes');
+  });
+
+  it('should show and focus on the username field when clicking on the edit icon', () => {
+    cy.findByTestId('creator-signup-form').as('creatorSignupForm');
+    cy.get('@creatorSignupForm')
+      .findByText(/^Name$/)
+      .type('Forem creator name');
+
+    cy.get('.js-creator-edit-username').click();
+    cy.get('@creatorSignupForm')
+      .findByText(/^Username/)
+      .should('exist');
+
+    cy.get('input[name="user[username]"]').should(
+      'have.value',
+      'forem_creator_name',
+    );
+  });
+
+  it('should contain an email label and field', () => {
+    cy.findByTestId('creator-signup-form').as('creatorSignupForm');
+    cy.get('@creatorSignupForm')
+      .findByText(/^Email/)
+      .should('be.visible');
+  });
+
+  it('should contain an password', () => {
+    cy.findByTestId('creator-signup-form').as('creatorSignupForm');
+    cy.get('@creatorSignupForm')
+      .findByText(/^Password/)
+      .should('be.visible');
+  });
+
+  it('should toggle the password when the eye icon is clicked', () => {
+    cy.findByTestId('creator-signup-form').as('creatorSignupForm');
+    cy.get('@creatorSignupForm')
+      .findByText(/^Password$/)
+      .type('abc123456');
+
+    cy.get('input[name="user[password]"]').should(
+      'have.attr',
+      'type',
+      'password',
+    );
+    cy.get('.js-eye').should('be.visible');
+    cy.get('.js-eye-off').should('not.be.visible');
+
+    cy.get('.js-creator-password-visibility').click({ force: true });
```

Comment:
I think we only need to force this because it's not a button 🤔

Reply:
It seemed that way, thanks and fixed!